gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -63,16 +63,24 @@
 """List of chromosomes in the combined FAF release."""
 
 
-def filter_gene_to_test(ht: hl.Table) -> hl.Table:
+def filter_gene_to_test(ht: hl.Table, pcsk9: bool, zfy: bool) -> hl.Table:
     """
-    Filter to PCSK9 1:55039447-55064852 for testing.
+    Filter to PCSK9 1:55039447-55064852 and/or ZFY Y:2935281-2982506 for testing.
 
     :param ht: Table with frequency and FAF information.
+    :param pcsk9: Whether to filter to PCSK9 1:55039447-55064852.
+    :param zfy: Whether to filter to ZFY Y:2935281-2982506.
     :return: Table with frequency and FAF information of the filtered interval of a gene
     """
+    filter_loci = []
+    if pcsk9:
+        filter_loci.append("chr1:55039447-55064852")
```

Comment:
Could you add a logger here? 
```suggestion
        logger.info("Filtering to a subset of variants on PCSK9 on chr1...")
        filter_loci.append("chr1:55039447-55064852")
```